R/select_best.R

```diff
@@ -38,7 +37,7 @@
 #' \donttest{
 #' data("example_ames_knn")
 #'
-#' show_best(ames_iter_search, metric = "rmse", maximize = FALSE)
+#' show_best(ames_iter_search, metric = "rmse")
```

Comment:
Can you add `...` and, if someone passes `maximize`, write out a message that it's no longer needed? I don't think that we need to use the lifecycle code for this since it is a minor function.

Reply:
This will involve moving the dots to the end of the function call for `select_by_pct_loss()` and `select_by_one_std_err()`, I believe. That's OK? Looks OK to me.

Reply:
Nope, never mind. Fine the way it is.